src/SplashScreen/SplashNav/SplashNavStyles.tsx

```diff
@@ -1,95 +1,67 @@
-.splash-header-container {
-  width: 80%;
-  display: flex;
-  justify-content: space-between;
-  margin: 0 auto;
-}
+import styled from '@emotion/styled';
 
-.splash-header {
-  background: white;
+export const Login = styled('a')`
+  border: 2px solid black;
+  transition: all 0.15s ease-in-out;
+  border-radius: 3.5px;
+  padding: 3px 15px;
+  transition: 300ms;
+  &:hover {
+  background-color: #18879b;
+  border: 2px solid #18879b;
+  
 }
-
-.splash-header::after {
-  content: '';
-  display: table;
-  clear: both;
+`;
+export const SignUp = styled('a')`
+   margin-left: -5px;
+  padding: 5px 17px;
+  border-radius: 3.5px;
+  background-color: #d6c20e;
+  &:hover {
+  opacity: 0.2;
 }
-
-.splash-logo {
+`;
+export const NavLink = styled('a')`
+  color: #131111;
+  text-decoration: none;
+  font-weight: bold;
+  bottom: -1.5px;
+  border-style: solid;
+  
+  transition: opacity 300ms;
+  &:hover {
+    opacity: 1;
+    border-width: 0 0 4px;
+    border-color: #0C6474;
+  }
+`;
+export const Img = styled('img')`
   float: left;
   padding: 20px 0;
-}
-
-.splash-nav {
+`;
+export const Container = styled('header')`
+  width: 80%;
+  display: flex;
+  justify-content: space-between;
+  margin: 0 auto;
+`;
+export const Header = styled('header')`
+  background: white;
+`;
+export const Nav = styled('nav')`
   padding-top: 1rem;
   display: flex;
-}
-
-.splash-nav ul {
+  color: #131111;
+  font-weight: bold;
+`;
+export const Ul = styled('ul')`
   margin: 0;
   margin: auto;
   padding: 0;
   list-style: none;
-}
-
-.splash-nav li {
+`;
+export const Li = styled('li')`
```

Comment:
I think these should become `NavItem` instead of `Li` because the styling on this component is specific to being a `NavItem` instead of just a list item. The component `Li` is not generic enough for just `Li` as a name.